### ISSUE TITLE
View in Coriolis doesn't work

### ISSUE THREAD

**Issue by Jibsman** [EDCD/Orbis#14]
The button "View in Coriolis" doesn't do anything. 
I thought I had already submitted this, but I don't see it anywhere.

**Comment by jessb321** [comment]
Will fix soontm assuming the issue is what I think it is

**Comment by Jibsman** [comment]
Still occurring.

**Comment by jessb321** [comment]
Yeah, I'm currently in the middle of a rather large restructure, which will have the fix.
